docs/source/ttnn/tt_metal_models/get_started.rst

```diff
@@ -10,12 +10,12 @@ Ensure that you have the base TT-Metalium source and environment configuration
 `built and ready
 <https://github.com/tenstorrent/tt-metal/blob/main/INSTALLING.md>`_.
 
-Now, from the project root, get the Python virtual environment in which you'll
-be working in ready.
+Now, from the project root, activate the provided Python virtual environment in
+which you'll be working.
 
 ::
 
-    source build/python_env/bin/activate
+    source python_env/bin/activate
```

Comment:
There's an env variable (`PYTHON_ENV_DIR`) that can change where this is. Maybe make a note of that here?

Reply:
Done, pls take a look